src/pages/signup/Second.tsx

```
@@ -0,0 +1,32 @@
+      <InputBox>
+        <Input type="password" name="password" value="" label="비밀번호" placeholder="⦁⦁⦁⦁⦁⦁⦁⦁" onChange={() => {}} />
+        <Input type="password" name="confirmPassword" value="" label="비밀번호 확인" placeholder="⦁⦁⦁⦁⦁⦁⦁⦁" onChange={() => {}} />
+      </InputBox>
+      <Button disabled bigSize onClick={() => {}}>
+        다음
+      </Button>
+    </>
+  )
+}
+
+const EmailInput = styled.div`
```

ESLint

[error] 'EmailInput' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)